src/panoptes/pocs/focuser/astromechanics.py

```diff
@@ -44,7 +44,8 @@ def position(self):
         """
         response = ''
         try:
-            response = int(self._send_command("P").rstrip("#"))
+            # Subtract the calibration position to get the actual position of the focuser.
+            response = int(self._send_command("P").rstrip("#")) - int(self._calibration_position)
```

Comment:
Shouldn't this be changed? I thought I did that in my review  🤔 

```suggestion
            response = int(self._send_command("P").rstrip("#")) - int(self._zero_position)
```